committee-steering/governance/annual-reports.md

```diff
@@ -21,83 +21,87 @@ should be highlighted
+owners or other community members
 - All questions are require a response. If something isn't applicable, n/a is
 fine.
-- This report will be considered private until Steering reviews and follows up
-with next steps of a pull request; however, please include any and all explicit
-private matters between [private] tags [/private]. We will absolutely not
-release information between private tags. If Chairs and/or Organizers would like
- to do the *Your Role* section individually and privately, please send to us at
- steering-private@kubernetes.io    
-- Use this [question template]
+- End users and other members of the community will read these. Err on the side
+of being more explict than using our upstream shorthand or abbreviations. 
+- If the report is submitted with private matters before the PR, until Steering 
+reviews and follows; however, please include any and all explicit
+private matters between [private] tags [/private].   
+- The [question template] can be forked into whatever medium you wish to
+collaborate with your community: gdocs, hackmd, etc - it will all land back into
+a pull request at the end. 
 
 ## Questions for report:
 
-#### You and your role:  
-When did you become a chair and do you enjoy the role?  
-What do you find challenging?  
-Do you have goals for the group?  
-Do you want to continue or find a replacement? If you feel that you aren’t ready
- to pass the baton, what would you like to accomplish before you do?  
-Is there something we can provide that would better support you?  
-Do you have feedback for Steering? Suggestions for what we should work on?  
-
 #### Special Interest Groups:
-- How are you doing with operational tasks in [yourgroup]-governance.md?
-- Is your README accurate? have a CONTRIBUTING.md file?
-- All subprojects correctly mapped and listed in sigs.yaml?
-- How does the group get updates, reports, or feedback from subprojects? Are
-there any that are being retired?
-- Are OWNERs files up to date?
-- Tell us about your meetings: large/small, active/quiet? learnings? up to date
-on the Kubernetes community YouTube under a community group playlist? Meeting
-notes up to date?
-- When was your last public community update? (provide link)
+##### Operational 
+- How are you doing with operational tasks in [SIG]-governance.md?
+  - Is your README accurate? have a CONTRIBUTING.md file?
+  - All subprojects correctly mapped and listed in sigs.yaml?
+  - What’s your meeting culture? Large/small, active/quiet, learnings? Meeting 
+  notes up to date? Are you keeping recordings up to date/trends in community 
+  members watching recordings?
+- How does the group get updates, reports, or feedback from subprojects? Are 
+there any springing up or being retired? Are OWNERS.md files up to date in these
+ areas?
+- Same question as above but for working groups.
+- When was your last monthly community-wide update? (provide link to deck and/or
```

Comment:
```suggestion
- When was your last public community update? (provide link to deck and/or
```
hm..I think the original language here is appropriate if we still want to have things like presentations at kubecon count.